### PR TITLE
QOL: add bash completion for p1 to localnet client

### PR DESCRIPTION
**Pull request by 0xBigBoss**
if you ever wondered if we could have tab completions inside localnet fret no more! This PR adds it to the cli client pod so you don't have to.

I looked at adding it to the docker setup, but that's not using the binary.

Enjoy! 👩🏻‍💻

```
make localnet_up # of course
make localnet_shell
root@dev-cli-client-9f4579876-5pmn7:~# p1 <TAB>
# Account  (Account specific commands)
# Application  (Application actor specific commands)
# Consensus  (Consensus specific commands)
# Fisherman  (Fisherman actor specific commands)
# Governance  (Governance specific commands)
# Keys  (Key specific commands)
# Query  (Commands related to querying on-chain data via the node's RPC server)
# Servicer  (Servicer specific commands)
# System  (Commands related to health and troubleshooting of the node instance)
# Validator  (Validator actor specific commands)
# completion  (Generate the autocompletion script for the specified shell)
# debug  (Debug utility for rapid development)
# help  (Help about any command)
# {"level":"debug","time":"2023-06-30T03:13:31Z","message":"Debug keybase initializing... Restoring from the embedded backup file..."}
```

### CODE REVIEW

File: build/localnet/Tiltfile [hunk 1]
```diff
@@ -120,8 +120,11 @@ docker_build_with_restart(
     "client-image",
     root_dir,
     dockerfile_contents="""FROM debian:bullseye
-RUN apt-get update && apt-get install -y procps
-WORKDIR /
+RUN apt-get update && apt-get install -y procps bash-completion
```
Comment (h5law) [inline build/localnet/Tiltfile:123]: Could we also are jq here? I find the output of RPC easier to grok when I can actually read it 😂

**Review by h5law — APPROVED** [review]
Small changes but big QoL impact! Love this will make debugging a lot nicer!

Left 1 comment on my personal preference to include jq as well. But LGTM 🚀

File: build/localnet/Tiltfile (hunk 1 above)
Comment (0xBigBoss) [reply build/localnet/Tiltfile:123]: i like that actually. I did find myself installing it lol

File: build/localnet/Tiltfile [hunk 2]
```diff
@@ -120,8 +120,11 @@ docker_build_with_restart(
     "client-image",
     root_dir,
     dockerfile_contents="""FROM debian:bullseye
-RUN apt-get update && apt-get install -y procps
-WORKDIR /
+RUN apt-get update && apt-get install -y procps bash-completion jq
+RUN echo "source /etc/bash_completion" >> ~/.bashrc
+# tail -n +2 removes the first line of the completion script since the CLI spits out some logs
+RUN echo "source <(p1 completion bash | tail -n +2)" >> ~/.bashrc
+WORKDIR /root
```
Comment (Olshansk) [inline build/localnet/Tiltfile:127]: Noticed we're changing the WORKDIR.

I know it's likely harmless but want to make sure that:

1. You're sure there's no harm
2. It was tested/validated E2E

Comment (Olshansk) [reply build/localnet/Tiltfile:127]: @okdas Can you TAL?

Comment (0xBigBoss) [reply build/localnet/Tiltfile:127]: Screenshot 2023-06-30 at 18 43 35

yup we should be GTG, not sure why workdir was set to `/`

Comment (Olshansk) [reply build/localnet/Tiltfile:127]: Works for me, feel free to merge!

**Pull request merged**